### PR TITLE
syncthing 0.13.1

### PR DESCRIPTION
**Pull request by mbi**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Install is failing, looks like missed packages: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1960/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_syncthing/

**Comment by mbi** [comment]
Bums. Sorry about that, I'll rise the issue upstream.

**Comment by coudenysj** [comment]
Looks like 0.13.1 came out an our ago.

**Comment by mbi** [comment]
@coudenysj ha! Okay, I'll update this PR.

**Comment by coudenysj** [comment]
Looks like the issues are resolved, too.

**Comment by mbi** [comment]
Indeed, it's fixed and the `upstream label` is no longer relevant either  /nudge @mikemcquaid 😏

File: Formula/syncthing.rb [hunk 1]
```diff
@@ -17,17 +17,17 @@ class Syncthing < Formula
   depends_on :hg => :build
 
   def install
-    ENV["GOPATH"] = cached_download/".gopath"
+    ENV["GOPATH"] = cached_download
```
Comment (mikemcquaid) [inline Formula/syncthing.rb]: Any idea why this wants built in the download path? Would be really nice to avoid that if we can.

**Comment by mbi** [comment]
@mikemcquaid okay, as per d1dc2c87e82f5fbcd082ea31ae4379b41fb8beec the formula builds in `cached_download/".gopath"` like the original one did. Is this better, or should it build completely outside of `cached_download`, e.g. `/tmp` ?

**Comment by mikemcquaid** [comment]
@mbi It would be good if it built in `buildpath` (which is the default location and a temporary directory). You can symlink in `.git` if necessary.

**Comment by mbi** [comment]
@mikemcquaid thank you, a8bb77177166756c674bc693c222e72f61e93d37 copies to `buildpath` and builds in there.

File: Formula/syncthing.rb [hunk 2]
```diff
@@ -17,15 +17,11 @@ class Syncthing < Formula
   depends_on :hg => :build
 
   def install
-    ENV["GOPATH"] = cached_download/".gopath"
+    ENV["GOPATH"] = buildpath/".syncthing-gopath"
+    mkdir_p buildpath/".syncthing-gopath/src/github.com/syncthing"
+    cp_r cached_download, buildpath/".syncthing-gopath/src/github.com/syncthing/syncthing"
```
Comment (mikemcquaid) [inline Formula/syncthing.rb]: I'm happy to accept this but just wondering if this can be symlinked instead of copied? If not: no worries.

Comment (mbi) [inline Formula/syncthing.rb]: @mikemcquaid as of syncthing 0.13.0 the build doesn't work with symlinks any longer, that's why it all broke down and the process had to be rewritten. [Here](https://github.com/syncthing/syncthing/issues/3106) is the relevant upstream discussion.

Comment (mikemcquaid) [inline Formula/syncthing.rb]: 🆗 🆒

**Comment by mikemcquaid** [comment]
Thanks again @mbi!

**Pull request closed without merging**